CONTRIBUTING.md

```diff
@@ -130,42 +151,49 @@ Every Python and Cython file should only include the following license header:
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ```
+
 The copyright year should be updated every calendar year.
 
 ## Security
 
 ### Bandit
 
-We use [Bandit](https://github.com/PyCQA/bandit) to find common security issues
-in Python code.
+We use [Bandit*](https://github.com/PyCQA/bandit) to find common security issues
+in the Python* code.
 
-Install: `pip install bandit`
+To install, run:
+```bash
+pip install bandit
+```
 
-- Revision: `1.7.0`
+Revision: `1.7.0`
 
-Run before each commit: `bandit -r dpctl -lll`
+Before each commit, run:
+```bash
+bandit -r dpctl -lll
+```
 
 ## Code Coverage
 
-Code coverage for both C and Python sources in dpctl is generated for each
+Code coverage, for both C and Python sources in dpctl, is generated for each
 pull request (PR). A PR cannot be merged if it leads to a drop in the code
-coverage by more than five percentage points. *Ergo, do not forget to write
-unit tests for your changes.* To check the code coverage for your code, follow
-these steps:
+coverage by more than five percentage points. Therefore, write
+unit tests for your changes. 
```

Comment:
```suggestion
coverage by more than five percentage points. Therefore, please supplement code changes with appropriate tests. 
```

Reply:
So we went from brash "*Ergo, do not forget to write unit tests for your changes.*" to formal "*Therefore, write
unit tests for your changes.*" to downright humble request "*Therefore, please supplement code changes with appropriate tests.*"  :laughing:

Reply:
@diptorupd Fair enough but we do not use "please" in the documentation. What about: 
```suggestion
Therefore, supplement code changes with appropriate tests.
```

Reply:
Ok, let's keep the original suggestion.